utils/get_supported_scylla_base_versions.py

```diff
@@ -11,6 +11,9 @@
 sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Please, use upper case for `LOGGER` variable.

Reply:
FIxed